docs/admin.md

```diff
@@ -0,0 +1,96 @@
+
+Cluster Administrator may want to customize Default Configuration due to internal preferences/limitations, for example:
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
+- Existed Taints and tolerations policy, so Backstage Pods have to be configured with certain tolerations restrictions.
+- ...
+
+Default Configuration is implemented as a ConfigMap called *backstage-default-config*, deployed on *backstage-system* namespace and mounted to Backstage controller container as a */default-config* directory.
+This config map contains the set of keys/values which maps to file names/contents in the */default-config*.
+These files contain yaml manifests of objects used by Backstage controller as an initial desired state of Backstage CR according to Backstage Operator configuration model:
+
+![bs-conf](https://github.com/gazarenkov/janus-idp-operator/assets/578124/d56cbbb0-781c-43fc-8624-8832893fede3)
+ 
+
+Mapping of configMap keys (yaml files) to runtime objects (NOTE: for the time (Dec 20'23) it is a subject of change):
+| Key/File name         | k8s/OCP Kind        | Mandatory*    |ver.|Notes                                      |
+| ----------------------|:-------------------:| -------------------|------------------------------------------:|
+| deployment.yaml       | appsv1.Deployment   | Yes           |all |Backstage deployment  |
+| service.yaml          | corev1.Service      | Yes           |all |Backstage Service |
+| db-statefulset.yaml   | appsv1.Statefulset  | For DB enabled|all |PostgreSQL statefulSet    |    
+| db-service.yaml       | corev1.Service      | For DB enabled|all |PostgreSQL Service   |
+| db-service-hl.yaml    | corev1.Service      | For DB enabled|all |PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled|all |Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  |all |Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            |0.02|Backstage app-config.yaml    |
+| configmap-files.yaml  | corev1.ConfigMap    | No            |0.02|Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            |0.02|Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            |0.02|Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            |0.02|Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            |0.02|dynamc-plugins config *    |
+| dynamic-plugins-      | corev1.ConfigMap    | No            |0.01|dynamc-plugins config *    |
+|configmap.yaml         |                     |               |    | |
+| backend-auth-         | corev1.ConfigMap    | No            |0.01|backend auth config    |
+|configmap.yaml         |                     |               |    | |
```

Comment:
The rendering of this table seems to have been lost with the latest changes:

![image](https://github.com/janus-idp/operator/assets/593208/f642d86a-10ce-47f6-aad3-cb9269f11310)

Can you fix it before t can be approved?

Reply:
Formatting fixed in [`ebb6c8f` (#64)](https://github.com/janus-idp/operator/pull/64/commits/ebb6c8f9045c9306729e2d912abee326b5c804bb)